File: 01_ProgrammingBasics/_01_FirstStepsInProgrammingMoreExercies/_04_VegetableMarket.java
package _01_ProgrammingBasics._01_FirstStepsInProgrammingMoreExercies;

import java.util.Scanner;

public class _04_VegetableMarket {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input prices and kilograms
        double priceForKilogramVegetables = Double.parseDouble(scan.nextLine());
        double priceForKilogramFruits = Double.parseDouble(scan.nextLine());
        int kilogramsVegetables = Integer.parseInt(scan.nextLine());
        int kilogramsFruits = Integer.parseInt(scan.nextLine());

        // Calculate total revenue
        double revenueInEuro = ((priceForKilogramVegetables * kilogramsVegetables) + (priceForKilogramFruits * kilogramsFruits)) / 1.94;

        // Print the total revenue
        System.out.printf("%.2f%n", revenueInEuro);

        scan.close();
    }
}

File: 23_C#Basics/_01_FirstStepsInProgrammingMoreExercises/_04_VegetableMarket/_04_VegetableMarket.cs
using System;

public class _04_VegetableMarket
{
    public static void Main(string[] args)
    {
        // Read input prices and kilograms
        double priceForKilogramVegetables = double.Parse(Console.ReadLine());
        double priceForKilogramFruits = double.Parse(Console.ReadLine());
        int kilogramsVegetables = int.Parse(Console.ReadLine());
        int kilogramsFruits = int.Parse(Console.ReadLine());

        // Calculate total revenue
        double revenueInEuro = ((priceForKilogramVegetables * kilogramsVegetables) + (priceForKilogramFruits * kilogramsFruits)) / 1.94;

        // Print the total revenue
        Console.WriteLine("{0:F2}", revenueInEuro);
    }
}
